src/MonitorProfiler/ProbeInstrumentation/ProbeInstrumentation.cpp

```diff
@@ -306,12 +309,15 @@ HRESULT STDMETHODCALLTYPE ProbeInstrumentation::GetReJITParameters(ModuleID modu
 {
     HRESULT hr;
 
+    m_pLogger->Log(LogLevel::Trace, _LS("ReJIT - moduleId: 0x%08x, methodDef: 0x%04x"), moduleId, methodDef);
```

Comment:
For any log statement that is in a hot path, especially uncoditioned ones, you should check if it's enabled for the level at which you want to log e.g. `if (m_pLogger->IsEnabled(LogLevel::Trace))`